noetic/distribution.yaml

```diff
@@ -2622,7 +2622,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/cubicleguy/ess_imu_driver-release.git
-      version: 1.0.1-3
+      version: 1.0.1-4
```

Comment:
I notice the [upstream repo `package.xml` says 2.0.0](https://github.com/cubicleguy/ess_imu_driver). Did you forget to create a tag the release?

Reply:
It seems this release 2.0.0 did not play well with the "bloom-release ess_imu_driver --rosdistro noetic" command.
I am actually not sure how to fix it either. Would welcome some advice.

Steps followed:
1. catkin_generate_changelog -> Commit change -> Seems ok
2. catkin_prepare_release --bump major -> Seems to update the package.xml correctly

...

> To https://github.com/cubicleguy/ess_imu_driver.git
>  * [new tag]         2.0.0 -> 2.0.0
> The source repository has been released successfully. The next step will be 'bloom-release'.

3. bloom-release ess_imu_driver --rosdistro noetic -> This is where it barfs
    Not sure why the script is creating --tag 1.0.1, see below

> bloom-release ess_imu_driver --rosdistro noetic
> ROS Distro index file associate with commit '38170796dd7bbacbd75d3f91d8feccb131ec8285'
> New ROS Distro index url: 'https://raw.githubusercontent.com/ros/rosdistro/38170796dd7bbacbd75d3f91d8feccb131ec8285/index-v4.yaml'
> ==> Fetching 'ess_imu_driver' repository from 'https://github.com/cubicleguy/ess_imu_driver-release.git'
> Cloning into '/tmp/tmpuofpqagl'...
> 
...

> Executing release track 'noetic'
> ==> bloom-export-upstream https://github.com/cubicleguy/ess_imu_driver.git git --tag 1.0.1 --display-uri https://github.com/cubicleguy/ess_imu_driver.git --name upstream --output-dir /tmp/tmp25k3iv3v
> Checking out repository at 'https://github.com/cubicleguy/ess_imu_driver.git' to reference '1.0.1'.
> Cloning into '/tmp/tc726yzj/upstream'...
...


Would you recommend I re-run the "bloom-release ess_imu_driver --rosdistro noetic" command?

- Raymond

Reply:
@cubicleguy Did you do git fetch and checkout the updated branch from which will be doing release e.g. noetic.
before step 3?
> 3. bloom-release ess_imu_driver --rosdistro noetic

Reply:
> Would you recommend I re-run the "bloom-release ess_imu_driver --rosdistro noetic" command?

Strange. I would recommend modifying [`tracks.yaml` on your release repo's `master` branch](https://github.com/cubicleguy/ess_imu_driver-release/blob/2f146efb49e11dc837b85eb1a2c45d27b4b959e4/tracks.yaml#L17C19-L17C23) so that `devel_branch` in the `noetic` track has the value `noetic` instead of `null` (because [your branch is called `noetic`](https://github.com/cubicleguy/ess_imu_driver/tree/noetic)):  and then try re-running the `bloom-release` command.